### PR TITLE
Still missing a lot of things, but this is just enough for a proof-of-concept Jupyter example

### PR DESCRIPTION
**Pull request by moshez**

### CODE REVIEW

File: examples/README.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This example uses sprites from
+https://kenney.nl/assets/animal-pac
```
Comment (pathunstrom) [inline examples/README.rst:2]: The best free resources.

File: examples/dynamic_penguin.ipynb [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    "Welcome to the interactive PursuedPyBear tutorial,\n",
+    "where we will learn to make games bit-by-bit.\n",
+    "PursuedPyBear (PPB) is designed to have little\n",
+    "unneeded code,\n",
+    "sometimes called\n",
+    "\"boilerplate\".\n",
+    "However, to tie together Jupyter and PPB,\n",
+    "we have to have a few uninteresting lines.\n",
+    "\n",
+    "Bear, um, with us.\n",
```
Comment (pathunstrom) [inline examples/dynamic_penguin.ipynb:18]: I love this line.

File: examples/dynamic_penguin.ipynb [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    "But slowly, so we can keep an eye on it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SPEED = 0.5\n",
```
Comment (pathunstrom) [inline examples/dynamic_penguin.ipynb]: Consider making this a class attribute ala

```suggestion
    "Penguin.speed = 0.5\n",
```

You can then use self.speed inside the new handler. (Not a strict suggestion, but closer to the best practices with ppb.)

File: txppb/__init__.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        from twisted.internet import reactor as _default_reactor
+        reactor = _default_reactor
+    setup = functools.partial(setup, reactor)
+    return ppb.make_engine(setup, starting_scene=starting_scene, title=title)
+
+
+@defer.inlineCallbacks
+def run(reactor=None,
+        setup: Callable[[Any, BaseScene], None]=None, *,
+        starting_scene=BaseScene, title="PursedPyBear"):
```
Comment (pathunstrom) [inline txppb/__init__.py]: Consider supporting (and passing along) **kwargs. If someone wants to use jupyter to build a subsystem, that's how initial configuration gets passed down in the core.

**Review by pathunstrom — COMMENTED** [review]
Some comments on the contents of the notebook itself, and one comment regarding the signature of the `run()` function. Very cool thing.

File: examples/dynamic_penguin.ipynb (hunk 3 above)
Comment (moshez) [reply examples/dynamic_penguin.ipynb]: Done

File: txppb/__init__.py (hunk 4 above)
Comment (moshez) [reply txppb/__init__.py]: Done.

**Pull request merged**